core/prometheus_exporter.go

```diff
@@ -0,0 +1,131 @@
+package core
+
+import (
+	"fmt"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	namespace = "gorb" // For Prometheus metrics.
+)
+
+var (
+	serviceHealth = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "service_health",
+		Help:      "Health of the load balancer service",
+	}, []string{"name", "host", "port", "protocol", "method", "persistent"})
```

Comment:
I think labels are meant to be used for multi-dimensional navigation, i.e. to specify metrics in an expression or for discovery. In this case, `method` and `persistence` do not add a dimension (because of the reasons I've outlined above) and thus do not carry a meaningful informative payload – that's the idea behind my comment.